server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -721,6 +722,21 @@ thruToken
    : (THROUGH | THRU)
    ;
 
+sqlDataTypesCaluse
+   : TYPE IS (sqlDataTypesCaluseForm1 | sqlDataTypesCaluseForm2)
+   ;
+
+sqlDataTypesCaluseForm1
+   : sqlDataTypes LPARENCHAR integerLiteral RPARENCHAR
```

Comment:
couldn't find VACHAR is option in the shared doc.  Also, there are some missing grammar for the SQL TYPE usage.

As mentioned by @VitGottwald , 
We should move these grammar to the sql preprocessor . 
Also, be mindful that adding this grammar to sql preprocessor, would mean that all variables and not just these specific cases are handled by sql preprocessor all the time ( when active).

For reference we can see following files from DaCo dialect :

https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/server/dialect-daco/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/VariableParser.g4

https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoCopybookVisitor.java

https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/2398ae361fbd44883fdcd2dd030d091f5fb4c217/server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoMaidProcessor.java#L276

Reply:
Discussed with @ishche , above statement might not be entirely correct. dialect can just handle their bit and that should be ok.